NDTensors/src/dense/set_types.jl

```diff
@@ -11,23 +11,8 @@ function set_datatype(storagetype::Type{<:Dense}, datatype::Type{<:AbstractArray
   )
 end
 
-SetParameters.unspecify_parameters(::Type{<:Dense}) = Dense
+# TypeParameterAccessors.unspecify_parameters(::Type{<:Dense}) = Dense
 
-SetParameters.parenttype_position(::Type{<:Dense}) = Position(2)
-SetParameters.nparameters(::Type{<:Dense}) = Val(2)
-SetParameters.get_parameter(::Type{<:Dense{P1}}, ::Position{1}) where {P1} = P1
-SetParameters.get_parameter(::Type{<:Dense{<:Any,P2}}, ::Position{2}) where {P2} = P2
-SetParameters.default_parameter(::Type{<:Dense}, ::Position{1}) = Float64
-SetParameters.default_parameter(::Type{<:Dense}, ::Position{2}) = Vector
-
-SetParameters.set_parameter(::Type{<:Dense}, ::Position{1}, P1) = Dense{P1}
-function SetParameters.set_parameter(
-  ::Type{<:Dense{<:Any,P2}}, ::Position{1}, P1
-) where {P2}
-  return Dense{P1,P2}
-end
-
-SetParameters.set_parameter(::Type{<:Dense}, ::Position{2}, P2) = Dense{<:Any,P2}
-function SetParameters.set_parameter(::Type{<:Dense{P1}}, ::Position{2}, P2) where {P1}
-  return Dense{P1,P2}
-end
+TypeParameterAccessors.parenttype_position(::Type{<:Dense}) = Position(2)
```

Comment:
```suggestion
TypeParameterAccessors.parenttype_position(::Type{<:Dense}) = 2
```

Reply:
I have changed this. But maybe we need to discuss the next line: i.e. how do we define `default_parameter` moving forward when position is an `Int`